shortcircuit-api/src/main/java/com/powsybl/shortcircuit/FeederResult.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.shortcircuit;
+
+/**
+ * @author Coline Piloquet <coline.piloquet at rte-france.com>
+ */
+public class FeederResult {
+
+    /**
+     * The ID of the connectable contributing to the three phase short circuit current
+     */
+    private final String connectableId;
+
+    /**
+     * The value of the current of the connectable contributing to the total fault current in kA
+     */
+    private final double feederThreePhaseCurrent;
+
+    public FeederResult() {
+        this("", 0);
```

Comment:
Just by curiosity, why is it useful? For serialization purposes?

Reply:
Yes! The serialization doesn't work without this constructor.